apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1219 @@
+import {
+  Box,
+  ActionCategoryCard,
+  ContentBlock,
+  Icon,
+  Inline,
+  Tag,
+  Text,
+  CTAProps,
+  Button,
+  GridContainer,
+  GridColumn,
+  GridRow,
+  Stack,
+  Checkbox,
+  Accordion,
+  AccordionItem,
+  Pagination,
+  LinkV2,
+  Input,
+  VisuallyHidden,
+  ToastContainer,
+  toast,
+  Hidden,
+  Filter,
+  FilterMultiChoice,
+} from '@island.is/island-ui/core'
+import { Screen } from '@island.is/web/types'
+import { withMainLayout } from '@island.is/web/layouts/main'
+import { useEffect, useRef, useState } from 'react'
+import * as styles from './UniversitySearch.css'
```

Comment:
Since this is a new file it'd be nice if the imports were a but more organized. Keeping the external libs at the top, followed by @island.is imports and finally having relative imports at the bottom